mmdeploy/codebase/mmdet/models/dense_heads/base_dense_head.py

```diff
@@ -80,7 +80,8 @@ def base_dense_head__get_bbox(ctx,
         ]
         mlvl_score_factors = []
     assert img_metas is not None
-    img_shape = img_metas[0]['img_shape']
+    img_shape = torch.tensor(
```

Comment:
since we have rewriting for `distance2bbox`, maybe we don't need to change here.